app/components/aircraft-queue.js

```diff
@@ -0,0 +1,82 @@
+import Ember from 'ember';
+const {computed} = Ember;
+import {TYPES as AIRCRAFT_TYPES, SIZES as AIRCRAFT_SIZES} from '../models/aircraft';
+
+// poor man's method of generating unique numbers,
+// purely to assist human readers in identifying airplanes
+const numbers = [];
+function makeNumber() {
```

Comment:
Aaaaah that makes more sense than "random open source do-gooder/ web crypto enthusiast"